addons/medical/Prefabs/Characters/Core/Character_Base.et

```diff
@@ -2,22 +2,475 @@ SCR_ChimeraCharacter {
  ID "520EC961A090B1EE"
  components {
   ActionsManagerComponent "{520EA1D2F659CE02}" {
+   ActionContexts {
+    UserActionContext "{5A26F8E7D24ECC4F}" {
+    }
+    UserActionContext "{61DD27FE10BC17E4}" {
+     UIInfo SCR_ActionContextUIInfo "{61DD27FE2603346C}" {
+      m_sLayoutName "{084B1DFFBEFA9555}UI/layouts/HUD/InteractionSystem/ContextBleedingInteractionBlip.layout"
+     }
+     ContextName "HeadBleeding"
+     Position PointInfo "{61DD27F9D631F1CB}" {
+      PivotID "Head"
+     }
+     LineOfSightCheck 1
+    }
+    UserActionContext "{61DD27CF88DEAF61}" {
+     UIInfo SCR_ActionContextUIInfo "{61DD27CF9942B52C}" {
+      m_sLayoutName "{7C1C535E08C5F282}UI/layouts/HUD/InteractionSystem/ContextHealInteractionBlip.layout"
+     }
+     ContextName "HeadHeal"
+     Position PointInfo "{61DD27CF93F496FC}" {
+      PivotID "Head"
+     }
+     LineOfSightCheck 1
+    }
+    UserActionContext "{61DD27F94D7CC578}" {
+     UIInfo SCR_ActionContextUIInfo "{61DD27F932464EAD}" {
+      m_sLayoutName "{084B1DFFBEFA9555}UI/layouts/HUD/InteractionSystem/ContextBleedingInteractionBlip.layout"
+     }
+     ContextName "ChestBleeding"
+     Position PointInfo "{61DD27F95C86684D}" {
+      PivotID "Spine5"
+      Offset 0.1 0.15 0
+     }
+     LineOfSightCheck 1
+    }
+    UserActionContext "{61DD27CF8E11D95F}" {
+     UIInfo SCR_ActionContextUIInfo "{61DD27CF77DAA241}" {
+      m_sLayoutName "{7C1C535E08C5F282}UI/layouts/HUD/InteractionSystem/ContextHealInteractionBlip.layout"
+     }
+     ContextName " ChestHeal"
```

Comment:
The typo in ContextName needs fixing.

Reply:
Oh, you mean the space before "ChestHeal"?

Reply:
Yup, I am not sure if it will break anything, but better safe than sorry.

Reply:
I don't think it would effect anything since the actionUI is connected to it via a drop down menu in the engine, I will fix it anyways for cleanliness. Thank you and good catch.The project is a WIP ofcourse and open to suggestions and refinement. Appreciate all input.